manager/assets/modext/widgets/security/modx.grid.access.policy.template.js

```diff
@@ -112,22 +113,37 @@ MODx.grid.AccessPolicyTemplate = function(config) {
             }]
         },'->',{
             xtype: 'textfield'
-            ,name: 'search'
-            ,id: 'modx-policy-template-search'
+            ,name: 'query'
```

Comment:
This was done initially in the first PR, not by me, I just bring it to one style.
The `query` is passed to the url, so if you rename it, it will be clearer, it seems to me. In general `name` is not used in the code at all.